translations/README_ar_SA.md

```diff
@@ -1,7 +1,20 @@
-﻿⚠️Note: This is a translated version of the original [README.md](https://github.com/Atlas-OS/Atlas/blob/main/README.md), information here may not be accurate and can be outdated.
```

Comment:
you do not delete this. Follow the [Translation Guidelines](https://github.com/Atlas-OS/Atlas/blob/main/translations/README.md).

Reply:
reverted